### PR TITLE
Add Enough Time guideline to accessibility recommendations

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Adds the [Enough Time](https://www.w3.org/TR/WCAG21/#enough-time) section to the list of WCAG 2.1 guidelines that we conform to.

## Why?

Enough Time includes some important guidance around:

- providing the user with an adequate amount of time to complete an action
- stopping, pausing or hiding non-essential animation

The latter recently came up in a discussion around a looping animation that appeared in a banner. Providing specific guidance on animation will help inform contributors on best practice in the future.

### CODE REVIEW

**Review by mbarton — APPROVED** [review]

File: accessibility.md [hunk 1]
```diff
@@ -9,12 +9,14 @@ Aim to build user interfaces that:
 - Provide [text alternatives](https://www.w3.org/TR/WCAG21/#text-alternatives)
 - Are [adaptable](https://www.w3.org/TR/WCAG21/#adaptable)
 - Are [distinguishable](https://www.w3.org/TR/WCAG21/#distinguishable)
+- Give [enough time][enough time] to read and use
```
Comment (mxdvl) [inline accessibility.md:12]:
```suggestion
- Give [enough time][] to read and use
```

File: accessibility.md [hunk 2]
```diff
@@ -9,12 +9,14 @@ Aim to build user interfaces that:
...
 - Do not [trigger seizures or physical reactions](https://www.w3.org/TR/WCAG21/#seizures-and-physical-reactions)
 - Are [navigable](https://www.w3.org/TR/WCAG21/#navigable)
 - Are [predictable](https://www.w3.org/TR/WCAG21/#navigable)
 - Help users [avoid or correct mistakes](https://www.w3.org/TR/WCAG21/#input-assistance)
 - Are [compatible][compatible] with existing or future users agents and assistive technologies
```
Comment (mxdvl) [inline accessibility.md:17]:
```suggestion
- Are [compatible][] with existing or future users agents and assistive technologies
```

**Review by mxdvl — APPROVED** [review]
Yes!⏳⌛

File: accessibility.md (hunk 1 above)
Comment (SiAdcock) [reply accessibility.md:12]: That is useful to know, thanks @mxdvl!

File: accessibility.md [hunk 3]
```diff
@@ -9,12 +9,14 @@ Aim to build user interfaces that:
...
 
+[enough time]: https://www.w3.org/TR/WCAG21/#enough-time
```
[the remark this answers is not in the document]
Comment (SiAdcock) [reply accessibility.md:19]: It's something @mxdvl recommended to make the markdown easier to read. Inline links interrupt the sentence, which is fine if the link is at the end of the sentence. If the link is in the middle of the sentence, we can extract it using this [reference-style link notation](https://www.markdownguide.org/basic-syntax/#reference-style-links).

[the remark this answers is not in the document]
Comment (mxdvl) [reply accessibility.md:19]: I think it's more _accessible_ this way 😉

**Pull request merged**